store-implementation/accumulo-store/src/test/java/uk/gov/gchq/gaffer/accumulostore/key/core/AbstractCoreKeyAccumuloElementConverterTest.java

```diff
@@ -60,53 +58,76 @@ public void shouldReturnOverriddenSerialiseNull() throws Exception {
                 .build();
 
         //when
-        new TestImplementation(testSchema).serialiseSizeAndPropertyValue("invalidProp", testSchema.getElement("group1"), new Properties(), stream);
+        createConverter(testSchema).serialiseSizeAndPropertyValue("invalidProp", testSchema.getElement("group1"), new Properties(), stream);
 
         //then
         assertArrayEquals(expected, stream.toByteArray());
     }
 
-    private class UnusualTestSerialiser extends StringSerialiser {
-        @Override
-        public byte[] serialiseNull() {
-            return "Empty".getBytes();
-        }
+    @Test
+    public void shouldDeserialiseSourceDestinationValuesCorrectWayRound() {
+        // Given 
+        final Edge edge = new Edge.Builder()
+                .source("1")
+                .dest("2")
+                .directed(true)
+                .group(TestGroups.ENTITY)
+                .build();
+
+        final byte[] rowKey = converter.getRowKeysFromEdge(edge).getFirst();
+        final byte[][] sourceDestValues = new byte[2][];
+
+        // When
+        final AbstractCoreKeyAccumuloElementConverter.EdgeDirection direction = converter.getSourceAndDestinationFromRowKey(rowKey, sourceDestValues);
+
+        // Then
+        assertEquals(AbstractCoreKeyAccumuloElementConverter.EdgeDirection.DIRECTED, direction);
     }
 
-    private class TestImplementation extends AbstractCoreKeyAccumuloElementConverter {
-        public TestImplementation(Schema schema) {
-            super(schema);
-        }
+    @Test
+    public void shouldDeserialiseSourceDestinationValuesIncorrectWayRound() {
+        // Given 
```

Comment:
Is this an extra character that has crept in? I've seen a few of these dotted around the test classes.